lib/handlers/delete.js

```diff
@@ -16,9 +16,7 @@ function handler(req, res) {
     ldp.delete(filename, function(err) {
         if (err) {
             debug("DELETE -- error: " + err);
-            return res
-                .status(err.status)
-                .send(err.message);
+            return next(err);
```

Comment:
I will have a look at this before the meeting, I could be wrong, but I have been reasoning about this, and I think yours is better design, since we always pass a next.